http-gateway/uri/uri.go

```diff
@@ -43,14 +43,13 @@ const (
 	APIWS string = API + "/ws"
 
 	// ocfcloud configuration
-	ClientConfiguration = "/.well-known/ocfcloud-configuration"
+	CloudConfiguration = "/.well-known/ocfcloud-configuration"
```

Comment:
Let's rename it to cloud-configuration instead of ocfcloud-configuration

Reply:
Renamed